src/main/java/ru/skypro/homework/service/mapping/AdMapping.java

```diff
@@ -0,0 +1,19 @@
+package ru.skypro.homework.service.mapping;
+
+import org.mapstruct.Mapper;
+import ru.skypro.homework.dto.Ad;
+import ru.skypro.homework.dto.CreateOrUpdateAd;
+import ru.skypro.homework.dto.ExtendedAd;
+
+@Mapper
+public interface AdMapping {
+
+    public Ad AdEntityToAdDto(ru.skypro.homework.model.Ad ad);
```

Comment:
```suggestion
    public Ad adEntityToAdDto(ru.skypro.homework.model.Ad ad);
```
названия методов должны начинаться с маленькой буквы